django_ipam/templates/admin/django-ipam/ip_address/change_form.html

```diff
@@ -0,0 +1,50 @@
+{% extends "admin/change_form.html" %}
+{% load static %}
+
+{% block admin_change_form_document_ready %}
+  {{ block.super }}
+  <script type="text/javascript">
+  (function ($) {
+      'use strict';
+      function getAvailableIp() {
+          var e = document.getElementById("id_subnet");
+          var subnet = e.options[e.selectedIndex].value;
+          $.ajax({
+              type: 'GET',
+              url: '{{ get_first_available_ip_url }}'.replace('0000', subnet),
```

Comment:
put back all this JS into a separate file so it will be cached by the browser, replace this line with:
```javascript
url: django.ipamGetFirstAvailableUrl.replace('0000', subnet),
```